packages/itwinui-react/src/core/utils/components/Divider.tsx

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../hooks';
+import '@itwin/itwinui-css/css/utils.css';
+
+export type DividerProps = React.ComponentPropsWithRef<'hr'>;
+
+/**
+ * Shows a divider
```

Comment:
If anyone has a better way to describe Divider, I'm all ears

Reply:
"element that helps to separate pieces of content into clear sections or parts."